package.json

```diff
@@ -15,11 +15,12 @@
   ],
   "scripts": {
     "build": "rimraf dist && NODE_ENV='production' webpack --optimize-minimize --progress",
-    "build-travis": "yarn build && node scripts/update-hashes.js",
+    "build:app": "BUILD_MODE='app' yarn build",
```

Comment:
@brucebinary could you please use https://www.npmjs.com/package/cross-env just to stop yarn from choking when used in windows environment.

I know @MahboobehMohammadi is on windows.